test/parsing_inputs.jl

```diff
@@ -0,0 +1,50 @@
+using LoopVectorization, Test, ArrayInterface
+
+# macros for generate loops whose body is not a block
+macro gen_loop_issue395(ex)
+  sym, ind = ex.args
+  loop_body = :(ret[$ind] = $sym[$ind])
+  loop = Expr(:for, :($ind = axes($sym, 1)), loop_body)
+  return esc(:(@avx $loop))
```

Comment:
`@turbo` is preferred for forward-compatibility. 
0.13 will deprecate `@avx`, and 0.14 will remove it. 

Although, there are no plans to actually release a 0.13 anytime soon. 
I don't see the point in breaking anything.